content/blog/pulumi-bloggers-oct-2019/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Four Infrastructure as Code Blog Posts You Should Read"
+authors: ["sophia-parafina"]
+tags: ["Pulumi","bloggers",]
+meta_image: "blog.png"
+meta_desc: "Continuous delivery of Pulumi apps with Octopus Deploy "
+date: "2019-10-24"
+---
+
+We are always excited when people join the Infrastructure as Code community and write about their experiences. Here are four new posts that we think are great!
```

Comment:
Is there any way to edit the preview? The preview seems to go into the first section which is a tad strange.

![image](https://user-images.githubusercontent.com/8304499/67536790-a163ba80-f68d-11e9-92e8-59b9e80d3052.png)

Reply:
Unsure why the preview looks like that.